examples.txt

```diff
@@ -157,6 +157,7 @@ Objects
 Object Initializer
 Object Member Functions
 Object Assignability
+Abstract Objects
```

Comment:
We need to add the new example to travis config as well.

Reply:
updated